src/app/clients/form/formSections/ListTypeCard.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import { errorText, valueOnChangeDropdownList } from "@/components/Form/formFunctions";
+import GenericFormCard from "@/components/Form/GenericFormCard";
+import { ErrorText } from "@/components/Form/formStyling";
+import DropdownListInput from "@/components/DataInput/DropdownListInput";
+import { ClientCardProps } from "../ClientForm";
+import { LIST_TYPES_ARRAY } from "@/common/fetch";
+import { capitaliseWords } from "@/common/format";
+
+type ListTypeLabelsAndValues = [string, string][];
+
+const ListTypeCard: React.FC<ClientCardProps> = ({
+    formErrors,
+    errorSetter,
+    fieldSetter,
+    fields,
+}) => {
+    const listTypeLabelsAndValues: ListTypeLabelsAndValues = LIST_TYPES_ARRAY.map((listType) => [
+        capitaliseWords(listType),
+        listType,
+    ]);
+
+    return (
+        <GenericFormCard
+            title="List Type"
+            required={true}
+            text="Which list should be used by default to pack parcels for this client?"
+        >
+            <DropdownListInput
+                selectLabelId="list-type-select-label"
+                labelsAndValues={listTypeLabelsAndValues}
+                listTitle="List Type"
+                defaultValue={fields.listType ? fields.listType : ""}
```

Comment:
We can use the [nullish coalescing operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing) here (the other nice thing about using `Type | null`): 
`    defaultValue={fields.listType ?? ""}`